subprojects/core/src/integTest/groovy/org/gradle/api/ConfigurationOnDemandIntegrationTest.groovy

```diff
@@ -35,7 +36,7 @@ class ConfigurationOnDemandIntegrationTest extends AbstractIntegrationSpec {
         file("gradle.properties") << "org.gradle.configureondemand=true"
     }
 
-    @IgnoreIf({ GradleContextualExecuter.isParallel() }) //parallel mode hides incubating message
+    @Requires(IntegTestPreconditions.NotParallelExecutor) //parallel mode hides incubating message
```

Comment:
The `@Requires` has a `reason` field, why not use this instead of comments? Applies to the other sites as well.